docs/docs-source/docs/modules/get-started/pages/installing-cloudflow.adoc

```diff
@@ -0,0 +1,361 @@
+= Installing Cloudflow
+:toc:
+:toc-title: ON THIS PAGE
+:toclevels: 2
+
+include::ROOT:partial$include.adoc[]
+
+This guide shows how to install Cloudflow, using the `Helm` and `kubectl` command-line tools.
+It also shows how to install Kafka, Spark and Flink operators that integrate with Cloudflow.
+
+== Prerequisites
+
+=== CLI Tools
+
+Make sure you have the following prerequisites installed before continuing:
+
+- Helm, version 3 or later (check with `helm version`)
+- Kubectl 
+
+Before proceeding, make sure that `kubectl` is correctly installed and access the Kubernetes cluster to install Cloudflow.
```

Comment:
```suggestion
Before proceeding, make sure that `kubectl` is correctly installed and access the Kubernetes cluster to install Cloudflow. This can be done with the following command:

    kubectl version
```